### DISCUSSION TITLE
Projects using pydantic for configuration

### DISCUSSION THREAD

**Discussion by archiif** [pydantic/pydantic#3610] [General]
Any good examples of real-word projects out there using pydantic for its configuration/settings management? Would like to see how it is used beyond the simple examples in the manual.

**Comment by samuelcolvin** [comment]
Most of my fully fledged examples are in private repos.

See https://github.com/samuelcolvin/foxglove/blob/master/foxglove/settings.py for an example, but that class is still to be inherited.
